app/models/amusement_park.rb

```diff
@@ -1,3 +1,8 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
-end
\ No newline at end of file
+  has_many :mechanics, through: :rides
+
+  def mechanics_list
+    mechanics.distinct
+  end
```

Comment:
Whoot! Really good job adding another association to transfer the responsibility of doing the joins over to Rails.